# Icelandic translation for update-manager
# Copyright (c) 2006 Rosetta Contributors and Canonical Ltd 2006
# This file is distributed under the same license as the update-manager package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2006.
#
msgid ""
msgstr ""
"Project-Id-Version: update-manager\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2012-04-14 00:52+0000\n"
"PO-Revision-Date: 2012-03-20 09:37+0000\n"
"Last-Translator: Sigurpáll Sigurðsson <Icekhaos@hotmail.com>\n"
"Language-Team: Icelandic <is@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=n != 1;\n"
"X-Launchpad-Export-Date: 2012-04-17 13:41+0000\n"
"X-Generator: Launchpad (build 15099)\n"

#. TRANSLATORS: download size of small updates, e.g. "250 kB"
#: ../DistUpgrade/utils.py:399 ../UpdateManager/Core/utils.py:399
#, python-format
msgid "%(size).0f kB"
msgid_plural "%(size).0f kB"
msgstr[0] "%(size).0f KB"
msgstr[1] "%(size).0f KB"

#. TRANSLATORS: download size of updates, e.g. "2.3 MB"
#: ../DistUpgrade/utils.py:402 ../UpdateManager/Core/utils.py:402
#, python-format
msgid "%.1f MB"
msgstr "%.1f MB"

#. TRANSLATORS: %s is a country
#: ../DistUpgrade/distro.py:206 ../DistUpgrade/distro.py:434
#, python-format
msgid "Server for %s"
msgstr "Netþjónn fyrir %s"

#. More than one server is used. Since we don't handle this case
#. in the user interface we set "custom servers" to true and
#. append a list of all used servers
#: ../DistUpgrade/distro.py:224 ../DistUpgrade/distro.py:230
#: ../DistUpgrade/distro.py:246
msgid "Main server"
msgstr "Aðalnetþjónn"

#: ../DistUpgrade/distro.py:250
msgid "Custom servers"
msgstr "Sérsniðnir netþjónar"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:141
msgid "Could not calculate sources.list entry"
msgstr ""

#: ../DistUpgrade/DistUpgradeAptCdrom.py:244
msgid ""
"Unable to locate any package files, perhaps this is not a Ubuntu Disc or the "
"wrong architecture?"
msgstr ""
"Tölvan fann engar pakkaskrár, er þetta örugglega Ubuntu-diskur eða kannski "
"vitlausa gerðin?"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:287
msgid "Failed to add the CD"
msgstr "Gat ekki bætt við geisladisknum"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:288
#, python-format
msgid ""
"There was a error adding the CD, the upgrade will abort. Please report this "
"as a bug if this is a valid Ubuntu CD.\n"
"\n"
"The error message was:\n"
"'%s'"
msgstr ""
"Það kom upp villa við að bæta geisladisknum við, uppfærslan mun nú hætta. "
"Vinsamlegast tilkynntu þetta sem lús ef þetta er gildur Ubuntu-"
"geisladiskur.\n"
"\n"
"Villuskilaboðin voru:\n"
"‚%s‘"

#: ../DistUpgrade/DistUpgradeCache.py:147
msgid "Remove package in bad state"
msgid_plural "Remove packages in bad state"
msgstr[0] "Fjarlægja pakka sem er í slæmu ástandi"
msgstr[1] "Fjarlægja pakka sem eru í slæmu ástandi"

#: ../DistUpgrade/DistUpgradeCache.py:150
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Do you want to remove this package now "
"to continue?"
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archives can be found for them. Do you want to remove these packages "
"now to continue?"
msgstr[0] ""
"Pakkinn ‚%s‘ er í lélegu ástandi og það er mikilvægt að hann verði settur "
"aftur upp, en hann fannst ekki í neinu gangasafni. Viltu eyða honum og halda "
"áfram?"
msgstr[1] ""
"Pakkarnir ‚%s‘ eru í lélegu ástandi og það er mikilvægt að þeir verði settir "
"aftur upp, en þeir fundust ekki í neinu gangasafni. Viltu eyða þeim og halda "
"áfram?"

#. FIXME: not ideal error message, but we just reuse a
#. existing one here to avoid a new string
#: ../DistUpgrade/DistUpgradeCache.py:248
msgid "The server may be overloaded"
msgstr "Netþjónninn gæti verið undir miklu álagi"

#: ../DistUpgrade/DistUpgradeCache.py:361
msgid "Broken packages"
msgstr "Bilaðir pakkar"

#: ../DistUpgrade/DistUpgradeCache.py:362
msgid ""
"Your system contains broken packages that couldn't be fixed with this "
"software. Please fix them first using synaptic or apt-get before proceeding."
msgstr ""
"Kerfið þitt inniheldur bilaða pakka sem ekki var hægt að laga með þessum "
"hugbúnaði. Reyndu að laga þá með ‚synaptic‘ eða ‚apt-get‘ áður en lengra er "
"haldið."

#. FIXME: change the text to something more useful
#: ../DistUpgrade/DistUpgradeCache.py:679
#, python-format
msgid ""
"An unresolvable problem occurred while calculating the upgrade:\n"
"%s\n"
"\n"
" This can be caused by:\n"
" * Upgrading to a pre-release version of Ubuntu\n"
" * Running the current pre-release version of Ubuntu\n"
" * Unofficial software packages not provided by Ubuntu\n"
"\n"
msgstr ""
"Villa kom upp við að reikna út uppfærsluna:\n"
"%s\n"
"\n"
" Nokkrar ástæður geta ollið þessu:\n"
" * Þú ert að uppfæra yfir í prufuútgáfu af Ubuntu\n"
" * Þú ert að nota prufuútgáfu af Ubuntu\n"
" * Þú hefur náð í hugbúnaðarpakka sem eru ekki á vegum Ubuntu\n"
"\n"

#: ../DistUpgrade/DistUpgradeCache.py:689
msgid "This is most likely a transient problem, please try again later."
msgstr "Þetta er væntanlega tímabundið vandamál. Reyndu aftur síðar."

#: ../DistUpgrade/DistUpgradeCache.py:692
msgid ""
"If none of this applies, then please report this bug using the command "
"'ubuntu-bug update-manager' in a terminal."
msgstr ""
"Ef ekkert af þessu á við, þá vinsamlegast tilkynntu þessa lús með því að "
"nota skipunina 'ubuntu-bug update-manager' í útstöð."

#: ../DistUpgrade/DistUpgradeCache.py:697
#: ../UpdateManager/UpdateManager.py:1035
msgid "Could not calculate the upgrade"
msgstr "Gat ekki reiknað út uppfærsluna"

#: ../DistUpgrade/DistUpgradeCache.py:748
msgid "Error authenticating some packages"
msgstr "Villa kom upp við auðkenningu nokkra pakka"

#: ../DistUpgrade/DistUpgradeCache.py:749
msgid ""
"It was not possible to authenticate some packages. This may be a transient "
"network problem. You may want to try again later. See below for a list of "
"unauthenticated packages."
msgstr ""
"Það tókst ekki að staðfesta nokkra pakka, þetta gæti verið tímabundið "
"netvandamál- reyndu aftur síðar. Að neðan er listi yfir þá pakka sem ekki "
"tókst að staðfesta."

#: ../DistUpgrade/DistUpgradeCache.py:769
#, python-format
msgid ""
"The package '%s' is marked for removal but it is in the removal blacklist."
msgstr ""
"Gefin var skipun um að fjarlægja pakkann ‚%s‘  en það er bannað að fjarlægja "
"hann."

#: ../DistUpgrade/DistUpgradeCache.py:773
#, python-format
msgid "The essential package '%s' is marked for removal."
msgstr "Það á að fjarlægja pakkann ‚%s‘ en hann er ómissandi."

#: ../DistUpgrade/DistUpgradeCache.py:782
#, python-format
msgid "Trying to install blacklisted version '%s'"
msgstr "Reyni að setja upp svartlistaða útgáfu ‚%s‘"

#: ../DistUpgrade/DistUpgradeCache.py:900
#, python-format
msgid "Can't install '%s'"
msgstr "Gat ekki sett upp ‚%s‘"

#: ../DistUpgrade/DistUpgradeCache.py:901
msgid ""
"It was impossible to install a required package. Please report this as a bug "
"using 'ubuntu-bug update-manager' in a terminal."
msgstr ""
"Ómögulegt var að setja upp nauðsynlegan pakka. Vinsamlegast tilkynntu þetta "
"sem lús með því að nota 'ubuntu-bug update-manager' í útstöð."

#. FIXME: provide a list
#: ../DistUpgrade/DistUpgradeCache.py:912
msgid "Can't guess meta-package"
msgstr "Get ekki giskað á lýsipakka"

#: ../DistUpgrade/DistUpgradeCache.py:913
msgid ""
"Your system does not contain a ubuntu-desktop, kubuntu-desktop, xubuntu-"
"desktop or edubuntu-desktop package and it was not possible to detect which "
"version of Ubuntu you are running.\n"
" Please install one of the packages above first using synaptic or apt-get "
"before proceeding."
msgstr ""
"Kerfið þitt inniheldur hvorki pakkann ‚ubuntu-desktop‘, ‚kubuntu-desktop‘, "
"‚xubuntu-desktop‘ né ‚edubuntu-desktop‘- og ekki tókst að bera kennsl á "
"hvaða tegund af Ubuntu þú ert að nota.\n"
" Settu upp einn af áðurnefndum pökkum með því að nota Synaptic-pakkastjórann "
"eða með því að nota apt-get áður en þú heldur áfram."

#: ../DistUpgrade/DistUpgradeController.py:100
msgid "Reading cache"
msgstr "Les skyndiminni"

#: ../DistUpgrade/DistUpgradeController.py:210
msgid "Unable to get exclusive lock"
msgstr "Fékk ekki útilokunarlæsingu"

#: ../DistUpgrade/DistUpgradeController.py:211
msgid ""
"This usually means that another package management application (like apt-get "
"or aptitude) already running. Please close that application first."
msgstr ""
"Þetta þýðir líklega að annar pakkastjóri (eins og ‚apt-get‘ eða ‚aptitude‘) "
"sé í gangi. Vinsamlegast lokaðu því forriti fyrst."

#: ../DistUpgrade/DistUpgradeController.py:244
msgid "Upgrading over remote connection not supported"
msgstr "Uppfærslur yfir fjartengingu eru óstuddar"

#: ../DistUpgrade/DistUpgradeController.py:245
msgid ""
"You are running the upgrade over a remote ssh connection with a frontend "
"that does not support this. Please try a text mode upgrade with 'do-release-"
"upgrade'.\n"
"\n"
"The upgrade will abort now. Please try without ssh."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:259
msgid "Continue running under SSH?"
msgstr "Viltu halda áfram að nota SSH?"

#: ../DistUpgrade/DistUpgradeController.py:260
#, python-format
msgid ""
"This session appears to be running under ssh. It is not recommended to "
"perform a upgrade over ssh currently because in case of failure it is harder "
"to recover.\n"
"\n"
"If you continue, an additional ssh daemon will be started at port '%s'.\n"
"Do you want to continue?"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:274
msgid "Starting additional sshd"
msgstr "Ræsi annað sshd"

#: ../DistUpgrade/DistUpgradeController.py:275
#, python-format
msgid ""
"To make recovery in case of failure easier, an additional sshd will be "
"started on port '%s'. If anything goes wrong with the running ssh you can "
"still connect to the additional one.\n"
msgstr ""
"Til að auðvelda bata þá mun auka sshd keyra í tenginum ‚%s‘. Ef eitthvað fer "
"úrskeyðis við að keyra SSH þá er hægt að tengjast við hina tenginguna.\n"

#: ../DistUpgrade/DistUpgradeController.py:283
#, python-format
msgid ""
"If you run a firewall, you may need to temporarily open this port. As this "
"is potentially dangerous it's not done automatically. You can open the port "
"with e.g.:\n"
"'%s'"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:355
#: ../DistUpgrade/DistUpgradeController.py:400
msgid "Can not upgrade"
msgstr "Get ekki uppfært"

#: ../DistUpgrade/DistUpgradeController.py:356
#, python-format
msgid "An upgrade from '%s' to '%s' is not supported with this tool."
msgstr "Þú getur ekki uppfært frá ‚%s‘ yfir í ‚%s' með þessu forriti."

#: ../DistUpgrade/DistUpgradeController.py:365
msgid "Sandbox setup failed"
msgstr "Uppsetning sandkassa mistókst"

#: ../DistUpgrade/DistUpgradeController.py:366
msgid "It was not possible to create the sandbox environment."
msgstr "Það tókst ekki að búa til sandkassann."

#: ../DistUpgrade/DistUpgradeController.py:372
msgid "Sandbox mode"
msgstr "Sandkassahamur"

#: ../DistUpgrade/DistUpgradeController.py:373
#, python-format
msgid ""
"This upgrade is running in sandbox (test) mode. All changes are written to "
"'%s' and will be lost on the next reboot.\n"
"\n"
"*No* changes written to a system directory from now until the next reboot "
"are permanent."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:401
msgid ""
"Your python install is corrupted. Please fix the '/usr/bin/python' symlink."
msgstr ""
"Það er eitthvað að python uppsetningunni þinni. Reyndu að laga "
"‚/usr/bin/python‘-vísunina."

#: ../DistUpgrade/DistUpgradeController.py:427
msgid "Package 'debsig-verify' is installed"
msgstr "Pakkinn 'debsig-verify' er uppsettur"

#: ../DistUpgrade/DistUpgradeController.py:428
msgid ""
"The upgrade can not continue with that package installed.\n"
"Please remove it with synaptic or 'apt-get remove debsig-verify' first and "
"run the upgrade again."
msgstr ""
"Uppfærslan getur ekki haldið áfram á meðan þessi pakki er á tölvunni.\n"
"Fjarlægðu hann með synaptic pakkastjóranum eða með því að nota ‚apt-get "
"remove debsig-verify‘ fyrst og keyra uppfærsluna aftur."

#: ../DistUpgrade/DistUpgradeController.py:440
#, python-format
msgid "Can not write to '%s'"
msgstr "Getur ekki skrifað í '%s'"

#: ../DistUpgrade/DistUpgradeController.py:441
#, python-format
msgid ""
"Its not possible to write to the system directory '%s' on your system. The "
"upgrade can not continue.\n"
"Please make sure that the system directory is writable."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:452
msgid "Include latest updates from the Internet?"
msgstr "Viltu láta nýjar uppfærslur af netinu fylgja með?"

#: ../DistUpgrade/DistUpgradeController.py:453
msgid ""
"The upgrade system can use the internet to automatically download the latest "
"updates and install them during the upgrade.  If you have a network "
"connection this is highly recommended.\n"
"\n"
"The upgrade will take longer, but when it is complete, your system will be "
"fully up to date.  You can choose not to do this, but you should install the "
"latest updates soon after upgrading.\n"
"If you answer 'no' here, the network is not used at all."
msgstr ""
"Uppfærslukerfið getur notað netið til að sækja nýjar uppfærslur sjálfkrafa "
"og að setja þær upp á meðan á uppfærslunni stendur. Þetta er ráðlagt ef þú "
"hefur nettengingu.\n"
"\n"
"Það tekur lengri tíma ef þú sækir uppfærslur af netinu, en á móti verður "
"tölvan af nýjustu gerð að uppfærslunni lokinni. Þú getur valið um að gera "
"þetta ekki en þú ættir að uppfæra kerfið þegar samt sem áður.\n"
"Netið verður ekki notað ef þú velur að gera þetta ekki."

#: ../DistUpgrade/DistUpgradeController.py:673
#, python-format
msgid "disabled on upgrade to %s"
msgstr "óvirkt að uppfæra til %s"

#: ../DistUpgrade/DistUpgradeController.py:700
msgid "No valid mirror found"
msgstr "Enginn spegill fannst"

#: ../DistUpgrade/DistUpgradeController.py:701
#, python-format
msgid ""
"While scanning your repository information no mirror entry for the upgrade "
"was found. This can happen if you run a internal mirror or if the mirror "
"information is out of date.\n"
"\n"
"Do you want to rewrite your 'sources.list' file anyway? If you choose 'Yes' "
"here it will update all '%s' to '%s' entries.\n"
"If you select 'No' the upgrade will cancel."
msgstr ""

#. hm, still nothing useful ...
#: ../DistUpgrade/DistUpgradeController.py:721
msgid "Generate default sources?"
msgstr "Búa til sjálfgefnar uppsprettur?"

#: ../DistUpgrade/DistUpgradeController.py:722
#, python-format
msgid ""
"After scanning your 'sources.list' no valid entry for '%s' was found.\n"
"\n"
"Should default entries for '%s' be added? If you select 'No', the upgrade "
"will cancel."
msgstr ""
"Engin færsla fyrir ‚%s‘ fannst í ‚sources.list‘ skránni þinni.\n"
"\n"
"Viltu að tölvan bæti inn færslu fyrir ‚%s‘ eða viltu velja ‚Nei‘ og hætta "
"við uppfærsluna?"

#: ../DistUpgrade/DistUpgradeController.py:757
msgid "Repository information invalid"
msgstr "Ógildar upplýsingar um gagnahirslu"

#: ../DistUpgrade/DistUpgradeController.py:758
msgid ""
"Upgrading the repository information resulted in a invalid file so a bug "
"reporting process is being started."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:765
msgid "Third party sources disabled"
msgstr "Frumgögn frá þriðja aðila voru gerð óvirk"

#: ../DistUpgrade/DistUpgradeController.py:766
msgid ""
"Some third party entries in your sources.list were disabled. You can re-"
"enable them after the upgrade with the 'software-properties' tool or your "
"package manager."
msgstr ""
"Einhverjir innslættir þriðju aðila voru gerðir óvirkir í skránni "
"‚sources.list‘. Þú getur virkjað þá seinna með tólinu ‚software-properties‘ "
"eða með pakkastjóranum þínum."

#: ../DistUpgrade/DistUpgradeController.py:806
msgid "Package in inconsistent state"
msgid_plural "Packages in inconsistent state"
msgstr[0] "Pakkinn er í versnandi ásigkomulagi"
msgstr[1] "Pakkarnir eru í versnandi ásigkomulagi"

#: ../DistUpgrade/DistUpgradeController.py:809
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Please reinstall the package manually or "
"remove it from the system."
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archive can be found for them. Please reinstall the packages manually "
"or remove them from the system."
msgstr[0] ""
"Það þarf að setja pakkann ‚%s‘ upp aftur þar sem hann er í ójafnvægi en hann "
"fannst hins vegar ekki í gagnahirslunni. Þú verður því að setja hann upp "
"handvirkt eða fjarlægja af tölvunni."
msgstr[1] ""
"Það þarf að setja pakkana ‚%s‘ upp aftur þar sem þeir eru í ójafnvægi en "
"þeir fundust hins vegar ekki í gagnahirslunni. Þú verður því að setja þá upp "
"handvirkt eða fjarlægja af tölvunni."

#: ../DistUpgrade/DistUpgradeController.py:857
msgid "Error during update"
msgstr "Villa kom upp í uppfærslu"

#: ../DistUpgrade/DistUpgradeController.py:858
msgid ""
"A problem occurred during the update. This is usually some sort of network "
"problem, please check your network connection and retry."
msgstr ""
"Það kom upp villa við að uppfæra tölvuna. Þetta er oftast vandamál með netið "
"þannig að best væri að athuga nettenginguna og reyna aftur."

#. print "on_button_install_clicked"
#: ../DistUpgrade/DistUpgradeController.py:867
#: ../UpdateManager/UpdateManager.py:758
msgid "Not enough free disk space"
msgstr "Ekki er nóg pláss á tölvunni"

#: ../DistUpgrade/DistUpgradeController.py:868
#, python-format
msgid ""
"The upgrade has aborted. The upgrade needs a total of %s free space on disk "
"'%s'. Please free at least an additional %s of disk space on '%s'. Empty "
"your trash and remove temporary packages of former installations using 'sudo "
"apt-get clean'."
msgstr ""

#. calc the dist-upgrade and see if the removals are ok/expected
#. do the dist-upgrade
#: ../DistUpgrade/DistUpgradeController.py:897
#: ../DistUpgrade/DistUpgradeController.py:1664
msgid "Calculating the changes"
msgstr "Reikna út breytingar"

#. ask the user
#: ../DistUpgrade/DistUpgradeController.py:929
msgid "Do you want to start the upgrade?"
msgstr "Viltu hefja uppfærsluna?"

#: ../DistUpgrade/DistUpgradeController.py:995
msgid "Upgrade canceled"
msgstr "Það var hætt við uppfærsluna"

#: ../DistUpgrade/DistUpgradeController.py:996
msgid ""
"The upgrade will cancel now and the original system state will be restored. "
"You can resume the upgrade at a later time."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1002
#: ../DistUpgrade/DistUpgradeController.py:1136
msgid "Could not download the upgrades"
msgstr "Gat ekki sótt uppfærslurnar"

#: ../DistUpgrade/DistUpgradeController.py:1003
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. All files downloaded so far have been kept."
msgstr ""

#. FIXME: strings are not good, but we are in string freeze
#. currently
#: ../DistUpgrade/DistUpgradeController.py:1087
#: ../DistUpgrade/DistUpgradeController.py:1124
#: ../DistUpgrade/DistUpgradeController.py:1229
msgid "Error during commit"
msgstr "Villa við staðfestingu"

#. generate a new cache
#: ../DistUpgrade/DistUpgradeController.py:1089
#: ../DistUpgrade/DistUpgradeController.py:1126
#: ../DistUpgrade/DistUpgradeController.py:1268
msgid "Restoring original system state"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1090
#: ../DistUpgrade/DistUpgradeController.py:1105
#: ../DistUpgrade/DistUpgradeController.py:1127
msgid "Could not install the upgrades"
msgstr "Gat ekki sett upp uppfærslurnar"

#. invoke the frontend now and show a error message
#: ../DistUpgrade/DistUpgradeController.py:1095
msgid ""
"The upgrade has aborted. Your system could be in an unusable state. A "
"recovery will run now (dpkg --configure -a)."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1100
#, python-format
msgid ""
"\n"
"\n"
"Please report this bug in a browser at "
"http://bugs.launchpad.net/ubuntu/+source/update-manager/+filebug and attach "
"the files in /var/log/dist-upgrade/ to the bug report.\n"
"%s"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1137
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. "
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1217
msgid "Remove obsolete packages?"
msgstr "Viltu fjarlægja úrelta pakka?"

#: ../DistUpgrade/DistUpgradeController.py:1218
#: ../DistUpgrade/DistUpgrade.ui.h:8
msgid "_Keep"
msgstr "_Halda"

#: ../DistUpgrade/DistUpgradeController.py:1218
msgid "_Remove"
msgstr "_Fjarlægja"

#: ../DistUpgrade/DistUpgradeController.py:1230
msgid ""
"A problem occurred during the clean-up. Please see the below message for "
"more information. "
msgstr ""
"Það kom upp vandamál við að hreinsa til. Lestu skilaboðin fyrir neðan til að "
"fá frekari upplýsingar. "

#. FIXME: instead of error out, fetch and install it
#. here
#: ../DistUpgrade/DistUpgradeController.py:1306
msgid "Required depends is not installed"
msgstr "Það er ekki búið að setja upp nauðsynleg ákvæði"

#: ../DistUpgrade/DistUpgradeController.py:1307
#, python-format
msgid "The required dependency '%s' is not installed. "
msgstr "Það er ekki búið að setja upp ákvæðið ‚%s‘ sem er nauðsynlegt. "

#. sanity check (check for ubuntu-desktop, brokenCache etc)
#. then open the cache (again)
#: ../DistUpgrade/DistUpgradeController.py:1568
#: ../DistUpgrade/DistUpgradeController.py:1633
msgid "Checking package manager"
msgstr "Athuga pakkastjórann"

#: ../DistUpgrade/DistUpgradeController.py:1573
msgid "Preparing the upgrade failed"
msgstr "Undirbúningur fyrir uppfærslu mistókst"

#: ../DistUpgrade/DistUpgradeController.py:1574
msgid ""
"Preparing the system for the upgrade failed so a bug reporting process is "
"being started."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1588
msgid "Getting upgrade prerequisites failed"
msgstr "Það tókst ekki að útvega forsendur uppfærslunnar"

#: ../DistUpgrade/DistUpgradeController.py:1589
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"Additionally, a bug reporting process is being started."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1617
msgid "Updating repository information"
msgstr "Uppfæri upplýsingar um gagnahirslur"

#: ../DistUpgrade/DistUpgradeController.py:1624
msgid "Failed to add the cdrom"
msgstr "Gat ekki bætt við geisladisknum"

#: ../DistUpgrade/DistUpgradeController.py:1625
msgid "Sorry, adding the cdrom was not successful."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1653
msgid "Invalid package information"
msgstr "Ógildar pakkaupplýsingar"

#: ../DistUpgrade/DistUpgradeController.py:1654
#, python-format
msgid ""
"After your package information was updated the essential package '%s' can "
"not be found anymore so a bug reporting process is being started."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1670
#: ../DistUpgrade/DistUpgradeController.py:1723
msgid "Fetching"
msgstr "Sæki"

#: ../DistUpgrade/DistUpgradeController.py:1676
#: ../DistUpgrade/DistUpgradeController.py:1727
msgid "Upgrading"
msgstr "Uppfæri"

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:1681
#: ../DistUpgrade/DistUpgradeController.py:1729
#: ../DistUpgrade/DistUpgradeController.py:1736
#: ../DistUpgrade/DistUpgradeController.py:1747
msgid "Upgrade complete"
msgstr "Uppfærslu lokið"

#: ../DistUpgrade/DistUpgradeController.py:1682
#: ../DistUpgrade/DistUpgradeController.py:1730
#: ../DistUpgrade/DistUpgradeController.py:1737
msgid ""
"The upgrade has completed but there were errors during the upgrade process."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1689
msgid "Searching for obsolete software"
msgstr "Leita að úreltum hugbúnaði"

#: ../DistUpgrade/DistUpgradeController.py:1698
msgid "System upgrade is complete."
msgstr "Stýrikerfisuppfærslu er lokið."

#: ../DistUpgrade/DistUpgradeController.py:1748
msgid "The partial upgrade was completed."
msgstr "Takmarkaðri uppfærslu er lokið."

#: ../DistUpgrade/DistUpgradeQuirks.py:202
msgid "evms in use"
msgstr "‚evms‘ í notkun"

#: ../DistUpgrade/DistUpgradeQuirks.py:203
msgid ""
"Your system uses the 'evms' volume manager in /proc/mounts. The 'evms' "
"software is no longer supported, please switch it off and run the upgrade "
"again when this is done."
msgstr ""
"Kerfið þitt notar ‚evms‘-gagnamiðilsstjóra í ‚/proc/mounts‘-möppunni. "
"Hugbúnaðurinn ‚evms‘ er ekki lengur studdur, slökktu á honum og keyrðu "
"uppfærsluna aftur að því loknu."

#: ../DistUpgrade/DistUpgradeQuirks.py:496
msgid ""
"Your graphics hardware may not be fully supported in Ubuntu 12.04 LTS."
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:498
msgid ""
"The support in Ubuntu 12.04 LTS for your Intel graphics hardware is limited "
"and you may encounter problems after the upgrade. For more information see "
"https://wiki.ubuntu.com/X/Bugs/UpdateManagerWarningForI8xx Do you want to "
"continue with the upgrade?"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:520
#: ../DistUpgrade/DistUpgradeQuirks.py:548
#: ../DistUpgrade/DistUpgradeQuirks.py:575
msgid ""
"Upgrading may reduce desktop effects, and performance in games and other "
"graphically intensive programs."
msgstr ""
"Þá má vera að brellur, leikir og önnur forrit sem nota skjákortið mikið "
"virki ekki eins vel að uppfærslu lokinni."

#: ../DistUpgrade/DistUpgradeQuirks.py:524
#: ../DistUpgrade/DistUpgradeQuirks.py:552
msgid ""
"This computer is currently using the NVIDIA 'nvidia' graphics driver. No "
"version of this driver is available that works with your video card in "
"Ubuntu 10.04 LTS.\n"
"\n"
"Do you want to continue?"
msgstr ""
"Þessi tölva er sem stendur að notast við NVIDIA ‚nvidia‘-rekilinn fyrir "
"grafík. Það er ekki til nein útgáfa af reklinum sem virkar með skjákortinu "
"þínu í Ubuntu 10.04 LTS\n"
"\n"
"Viltu halda áfram?"

#: ../DistUpgrade/DistUpgradeQuirks.py:579
msgid ""
"This computer is currently using the AMD 'fglrx' graphics driver. No version "
"of this driver is available that works with your hardware in Ubuntu 10.04 "
"LTS.\n"
"\n"
"Do you want to continue?"
msgstr ""
"Þessi tölva er sem stendur að notast við AMD ‚fglrx‘-rekilinn fyrir grafík. "
"Það er ekki til nein útgáfa af þessum rekli sem virkar með vélbúnaði þínum í "
"Ubuntu 10.04 LTS.\n"
"\n"
"Viltu halda áfram?"

#: ../DistUpgrade/DistUpgradeQuirks.py:609
msgid "No i686 CPU"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:610
msgid ""
"Your system uses an i586 CPU or a CPU that does not have the 'cmov' "
"extension. All packages were built with optimizations requiring i686 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:646
msgid "No ARMv6 CPU"
msgstr "Enginn ARMv6-örgjörvi"

#: ../DistUpgrade/DistUpgradeQuirks.py:647
msgid ""
"Your system uses an ARM CPU that is older than the ARMv6 architecture. All "
"packages in karmic were built with optimizations requiring ARMv6 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""
"Tölva þín notar ARM-örgjörva sem er eldri en ARMv6 (útgáfa 6). Allir pakkar "
"í karmic þurfa að lágmarki örgjörva af útgáfu ARMv6. Það er ekki hægt að "
"uppfæra stýrikerfi þitt yfir í nýja útgáfu af Ubuntu með þessum vélbúnaði."

#: ../DistUpgrade/DistUpgradeQuirks.py:667
msgid "No init available"
msgstr "‚init‘ fannst ekki"

#: ../DistUpgrade/DistUpgradeQuirks.py:668
msgid ""
"Your system appears to be a virtualised environment without an init daemon, "
"e.g. Linux-VServer. Ubuntu 10.04 LTS cannot function within this type of "
"environment, requiring an update to your virtual machine configuration "
"first.\n"
"\n"
"Are you sure you want to continue?"
msgstr ""

#: ../DistUpgrade/DistUpgradeMain.py:63
msgid "Sandbox upgrade using aufs"
msgstr "Uppfærsla í sandkassanum með ‚aufs‘"

#: ../DistUpgrade/DistUpgradeMain.py:65
msgid "Use the given path to search for a cdrom with upgradable packages"
msgstr ""
"Nota þessa slóð til að leita að geisladiski með pökkum sem hægt er að "
"uppfæra með"

#: ../DistUpgrade/DistUpgradeMain.py:71
msgid ""
"Use frontend. Currently available: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"
msgstr ""
"Nota framenda. Nú þegar tiltækir: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"

#: ../DistUpgrade/DistUpgradeMain.py:74
msgid "*DEPRECATED* this option will be ignored"
msgstr ""

#: ../DistUpgrade/DistUpgradeMain.py:77
msgid "Perform a partial upgrade only (no sources.list rewriting)"
msgstr "Ekki uppfæra allt (mun ekki skrifa yfir ‚sources.list‘)"

#: ../DistUpgrade/DistUpgradeMain.py:80
msgid "Disable GNU screen support"
msgstr ""

#: ../DistUpgrade/DistUpgradeMain.py:82
msgid "Set datadir"
msgstr "Ákveða ‚datadir‘"

#. print "mediaChange %s %s" % (medium, drive)
#: ../DistUpgrade/DistUpgradeViewGtk.py:114
#: ../DistUpgrade/DistUpgradeViewGtk3.py:117
#: ../DistUpgrade/DistUpgradeViewKDE.py:193
#: ../UpdateManager/DistUpgradeFetcherKDE.py:148
#, python-format
msgid "Please insert '%s' into the drive '%s'"
msgstr "Settu ‚%s‘ í drifið ‚%s‘"

#: ../DistUpgrade/DistUpgradeViewGtk.py:135
#: ../DistUpgrade/DistUpgradeViewGtk3.py:138
#: ../DistUpgrade/DistUpgradeViewKDE.py:207
msgid "Fetching is complete"
msgstr "Búið að sækja"

#: ../DistUpgrade/DistUpgradeViewGtk.py:146
#: ../DistUpgrade/DistUpgradeViewGtk3.py:149
#: ../DistUpgrade/DistUpgradeViewKDE.py:220
#, python-format
msgid "Fetching file %li of %li at %sB/s"
msgstr "Sæki skrá %li af %li á %sB/sek."

#: ../DistUpgrade/DistUpgradeViewGtk.py:149
#: ../DistUpgrade/DistUpgradeViewGtk.py:295
#: ../DistUpgrade/DistUpgradeViewGtk3.py:152
#: ../DistUpgrade/DistUpgradeViewGtk3.py:301
#: ../DistUpgrade/DistUpgradeViewKDE.py:221
#: ../DistUpgrade/DistUpgradeViewKDE.py:365
#, python-format
msgid "About %s remaining"
msgstr "Um það bil %s eftir"

#: ../DistUpgrade/DistUpgradeViewGtk.py:152
#: ../DistUpgrade/DistUpgradeViewGtk3.py:155
#: ../DistUpgrade/DistUpgradeViewKDE.py:223
#, python-format
msgid "Fetching file %li of %li"
msgstr "Sæki skrá %li af %li"

#. FIXME: add support for the timeout
#. of the terminal (to display something useful then)
#. -> longer term, move this code into python-apt
#: ../DistUpgrade/DistUpgradeViewGtk.py:183
#: ../DistUpgrade/DistUpgradeViewGtk3.py:186
#: ../DistUpgrade/DistUpgradeViewKDE.py:257
msgid "Applying changes"
msgstr "Virki breytingar"

#. we do not report followup errors from earlier failures
#: ../DistUpgrade/DistUpgradeViewGtk.py:208
#: ../DistUpgrade/DistUpgradeViewGtk3.py:211
#: ../DistUpgrade/DistUpgradeViewKDE.py:270
msgid "dependency problems - leaving unconfigured"
msgstr "vandamál með ákvæði - skil eftir óstillt"

#: ../DistUpgrade/DistUpgradeViewGtk.py:213
#: ../DistUpgrade/DistUpgradeViewGtk3.py:216
#: ../DistUpgrade/DistUpgradeViewKDE.py:272
#, python-format
msgid "Could not install '%s'"
msgstr "Gat ekki sett upp ‚%s‘"

#: ../DistUpgrade/DistUpgradeViewGtk.py:214
#: ../DistUpgrade/DistUpgradeViewGtk3.py:217
#: ../DistUpgrade/DistUpgradeViewKDE.py:273
#, python-format
msgid ""
"The upgrade will continue but the '%s' package may not be in a working "
"state. Please consider submitting a bug report about it."
msgstr ""
"Uppfærslan mun halda áfram en pakkinn ‚%s‘ er líklega í ólagi. Vinsamlegast "
"sentu inn upplýsingar um þessa bilun."

#. self.expander.set_expanded(True)
#: ../DistUpgrade/DistUpgradeViewGtk.py:231
#: ../DistUpgrade/DistUpgradeViewGtk3.py:234
#: ../DistUpgrade/DistUpgradeViewKDE.py:294
#, python-format
msgid ""
"Replace the customized configuration file\n"
"'%s'?"
msgstr ""
"Skipta út sérsniðnu stillingarskráni\n"
" ‚%s‘?"

#: ../DistUpgrade/DistUpgradeViewGtk.py:232
#: ../DistUpgrade/DistUpgradeViewGtk3.py:235
#: ../DistUpgrade/DistUpgradeViewKDE.py:295
msgid ""
"You will lose any changes you have made to this configuration file if you "
"choose to replace it with a newer version."
msgstr ""
"Þú munt tapa öllu sem þú hefur breytt í þessari stillingarskrá ef þú ákveður "
"að skipta henni út fyrir nýja útgáfu."

#: ../DistUpgrade/DistUpgradeViewGtk.py:250
#: ../DistUpgrade/DistUpgradeViewGtk3.py:253
#: ../DistUpgrade/DistUpgradeViewKDE.py:317
msgid "The 'diff' command was not found"
msgstr "Skipunin ‚diff‘ fannst ekki"

#: ../DistUpgrade/DistUpgradeViewGtk.py:463
#: ../DistUpgrade/DistUpgradeViewGtk3.py:469
#: ../DistUpgrade/DistUpgradeViewText.py:90
msgid "A fatal error occurred"
msgstr "Það kom upp grafalvarleg villa"

#: ../DistUpgrade/DistUpgradeViewGtk.py:464
#: ../DistUpgrade/DistUpgradeViewGtk3.py:470
msgid ""
"Please report this as a bug (if you haven't already) and include the files "
"/var/log/dist-upgrade/main.log and /var/log/dist-upgrade/apt.log in your "
"report. The upgrade has aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""

#: ../DistUpgrade/DistUpgradeViewGtk.py:481
#: ../DistUpgrade/DistUpgradeViewGtk3.py:487
msgid "Ctrl-c pressed"
msgstr "Ýtt á ctrl-c"

#: ../DistUpgrade/DistUpgradeViewGtk.py:482
#: ../DistUpgrade/DistUpgradeViewGtk3.py:488
msgid ""
"This will abort the operation and may leave the system in a broken state. "
"Are you sure you want to do that?"
msgstr ""
"Þetta mun stöðva aðgerðina og skilja kerfið eftir bilað. Ertu viss um að þú "
"viljir halda áfram?"

#. append warning
#: ../DistUpgrade/DistUpgradeViewGtk.py:630
#: ../DistUpgrade/DistUpgradeViewGtk3.py:621
msgid "To prevent data loss close all open applications and documents."
msgstr "Til að fyrirbyggja tap á gögnum skaltu loka öllum forritum og skrám."

#: ../DistUpgrade/DistUpgradeViewGtk.py:644
#: ../DistUpgrade/DistUpgradeViewGtk3.py:635
#, python-format
msgid "No longer supported by Canonical (%s)"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewGtk.py:645
#: ../DistUpgrade/DistUpgradeViewGtk3.py:636
#, python-format
msgid "<b>Downgrade (%s)</b>"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewGtk.py:646
#: ../DistUpgrade/DistUpgradeViewGtk3.py:637
#, python-format
msgid "Remove (%s)"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewGtk.py:647
#: ../DistUpgrade/DistUpgradeViewGtk3.py:638
#, python-format
msgid "No longer needed (%s)"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewGtk.py:648
#: ../DistUpgrade/DistUpgradeViewGtk3.py:639
#, python-format
msgid "Install (%s)"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewGtk.py:649
#: ../DistUpgrade/DistUpgradeViewGtk3.py:640
#, python-format
msgid "Upgrade (%s)"
msgstr ""

#. change = QMessageBox.question(None, _("Media Change"), msg, QMessageBox.Ok, QMessageBox.Cancel)
#: ../DistUpgrade/DistUpgradeViewKDE.py:194
#: ../UpdateManager/DistUpgradeFetcherKDE.py:150
msgid "Media Change"
msgstr "Breyta miðli"

#: ../DistUpgrade/DistUpgradeViewKDE.py:329
msgid "Show Difference >>>"
msgstr "Sýna mun >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:332
msgid "<<< Hide Difference"
msgstr "<<< Fela mun"

#: ../DistUpgrade/DistUpgradeViewKDE.py:547
msgid "Error"
msgstr "Villa"

#: ../DistUpgrade/DistUpgradeViewKDE.py:559
msgid "&Cancel"
msgstr "&Hætta við"

#: ../DistUpgrade/DistUpgradeViewKDE.py:561
#: ../DistUpgrade/DistUpgradeViewKDE.py:803
msgid "&Close"
msgstr "&Loka"

#: ../DistUpgrade/DistUpgradeViewKDE.py:607
msgid "Show Terminal >>>"
msgstr "Sýna útstöðina >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:610
msgid "<<< Hide Terminal"
msgstr "<<< Fela útstöðina"

#: ../DistUpgrade/DistUpgradeViewKDE.py:690
msgid "Information"
msgstr "Upplýsingar"

#: ../DistUpgrade/DistUpgradeViewKDE.py:741
#: ../DistUpgrade/DistUpgradeViewKDE.py:786
#: ../DistUpgrade/DistUpgradeViewKDE.py:789 ../DistUpgrade/DistUpgrade.ui.h:7
msgid "Details"
msgstr "Frekari upplýsingar"

#: ../DistUpgrade/DistUpgradeViewKDE.py:767
#, python-format
msgid "No longer supported %s"
msgstr "Ekki lengur stutt %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:769
#, python-format
msgid "Remove %s"
msgstr "Fjarlægja %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:771
#: ../DistUpgrade/DistUpgradeViewText.py:179
#, python-format
msgid "Remove (was auto installed) %s"
msgstr "Fjarlægja (var sett upp sjálfvirkt) %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:773
#, python-format
msgid "Install %s"
msgstr "Setja upp %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:775
#, python-format
msgid "Upgrade %s"
msgstr "Uppfæra %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:799
#: ../DistUpgrade/DistUpgradeViewText.py:227
msgid "Restart required"
msgstr "Það þarf að endurræsa tölvuna"

#: ../DistUpgrade/DistUpgradeViewKDE.py:799
msgid "<b><big>Restart the system to complete the upgrade</big></b>"
msgstr "<b><big>Endurræsa kerfið til að ljúka uppfærslunni</big></b>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:802 ../DistUpgrade/DistUpgrade.ui.h:14
#: ../data/gtkbuilder/UpdateManager.ui.h:26
msgid "_Restart Now"
msgstr "_Endurræsa núna"

#. FIXME make this user friendly
#: ../DistUpgrade/DistUpgradeViewKDE.py:820
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly advised to resume the upgrade."
msgstr ""
"<b><big>Viltu hætta við uppfærsluna?</big></b>\n"
"\n"
"Það getur verið að tölvan þín verði ónothæf ef þú hættir núna. Það er "
"stranglega mælt með því að halda áfram."

#: ../DistUpgrade/DistUpgradeViewKDE.py:824
msgid "Cancel Upgrade?"
msgstr "Viltu hætta við uppfærsluna?"

#: ../DistUpgrade/DistUpgradeView.py:59
#, python-format
msgid "%li day"
msgid_plural "%li days"
msgstr[0] "%li dagur"
msgstr[1] "%li dagar"

#: ../DistUpgrade/DistUpgradeView.py:61
#, python-format
msgid "%li hour"
msgid_plural "%li hours"
msgstr[0] "%li klukkutími"
msgstr[1] "%li klukkutímar"

#: ../DistUpgrade/DistUpgradeView.py:63
#, python-format
msgid "%li minute"
msgid_plural "%li minutes"
msgstr[0] "%li mínúta"
msgstr[1] "%li mínútur"

#: ../DistUpgrade/DistUpgradeView.py:64
#, python-format
msgid "%li second"
msgid_plural "%li seconds"
msgstr[0] "%li sekúnda"
msgstr[1] "%li sekúndur"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_days)s %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#. 
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#. 
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:80
#, python-format
msgid "%(str_days)s %(str_hours)s"
msgstr "%(str_days)s %(str_hours)s"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#. 
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#. 
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:98
#, python-format
msgid "%(str_hours)s %(str_minutes)s"
msgstr "%(str_hours)s %(str_minutes)s"

#. 56 kbit
#. 1Mbit = 1024 kbit
#: ../DistUpgrade/DistUpgradeView.py:149
#, python-format
msgid ""
"This download will take about %s with a 1Mbit DSL connection and about %s "
"with a 56k modem."
msgstr ""
"Þetta niðurhal mun taka um %s með 1Mbit DSL tengingu og um %s með 56k "
"símalínutengingu."

#. if we have a estimated speed, use it
#: ../DistUpgrade/DistUpgradeView.py:153
#, python-format
msgid "This download will take about %s with your connection. "
msgstr "Þetta mun taka um %s með þinni tengingu. "

#: ../DistUpgrade/DistUpgradeView.py:255 ../DistUpgrade/DistUpgrade.ui.h:21
msgid "Preparing to upgrade"
msgstr "Undirbý að uppfæra"

#: ../DistUpgrade/DistUpgradeView.py:256
msgid "Getting new software channels"
msgstr "Sæki nýjar hugbúnaðarrásir"

#: ../DistUpgrade/DistUpgradeView.py:257 ../DistUpgrade/DistUpgrade.ui.h:23
msgid "Getting new packages"
msgstr "Sæki nýja pakka"

#: ../DistUpgrade/DistUpgradeView.py:258 ../DistUpgrade/DistUpgrade.ui.h:26
msgid "Installing the upgrades"
msgstr "Set upp uppfærslurnar"

#: ../DistUpgrade/DistUpgradeView.py:259 ../DistUpgrade/DistUpgrade.ui.h:25
msgid "Cleaning up"
msgstr "Hreinsa til"

#: ../DistUpgrade/DistUpgradeView.py:344
#, python-format
msgid ""
"%(amount)d installed package is no longer supported by Canonical. You can "
"still get support from the community."
msgid_plural ""
"%(amount)d installed packages are no longer supported by Canonical. You can "
"still get support from the community."
msgstr[0] ""
msgstr[1] ""

#. FIXME: make those two separate lines to make it clear
#. that the "%" applies to the result of ngettext
#: ../DistUpgrade/DistUpgradeView.py:353
#, python-format
msgid "%d package is going to be removed."
msgid_plural "%d packages are going to be removed."
msgstr[0] "%d pakki mun vera fjarlægður."
msgstr[1] "%d pakkar munu vera fjarlægðir."

#: ../DistUpgrade/DistUpgradeView.py:358
#, python-format
msgid "%d new package is going to be installed."
msgid_plural "%d new packages are going to be installed."
msgstr[0] "%d nýr pakki mun vera uppsettur."
msgstr[1] "%d nýir pakkar munu vera uppsettir."

#: ../DistUpgrade/DistUpgradeView.py:364
#, python-format
msgid "%d package is going to be upgraded."
msgid_plural "%d packages are going to be upgraded."
msgstr[0] "%d pakki mun vera uppfærður."
msgstr[1] "%d pakkar munu vera uppfærðir."

#: ../DistUpgrade/DistUpgradeView.py:369
#, python-format
msgid ""
"\n"
"\n"
"You have to download a total of %s. "
msgstr ""
"\n"
"\n"
"Þú þarft að sækja samtals %s. "

#: ../DistUpgrade/DistUpgradeView.py:374
msgid ""
"Installing the upgrade can take several hours. Once the download has "
"finished, the process cannot be canceled."
msgstr ""

#: ../DistUpgrade/DistUpgradeView.py:378
msgid ""
"Fetching and installing the upgrade can take several hours. Once the "
"download has finished, the process cannot be canceled."
msgstr ""

#: ../DistUpgrade/DistUpgradeView.py:383
msgid "Removing the packages can take several hours. "
msgstr ""

#. FIXME: this should go into DistUpgradeController
#: ../DistUpgrade/DistUpgradeView.py:388 ../UpdateManager/UpdateManager.py:683
msgid "The software on this computer is up to date."
msgstr ""

#: ../DistUpgrade/DistUpgradeView.py:389
msgid ""
"There are no upgrades available for your system. The upgrade will now be "
"canceled."
msgstr ""
"Það eru engar uppfærslur fáanlegar fyrir kerfið þitt. Uppfærslu verður nú "
"hætt."

#: ../DistUpgrade/DistUpgradeView.py:402
msgid "Reboot required"
msgstr "Þú þarft að endurræsa tölvuna"

#: ../DistUpgrade/DistUpgradeView.py:403
msgid ""
"The upgrade is finished and a reboot is required. Do you want to do this now?"
msgstr ""
"Uppfærslum er lokið og nú þarf að endurræsa tölvuna. Viltu gera það núna?"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:70
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:70
#, python-format
msgid "authenticate '%(file)s' against '%(signature)s' "
msgstr ""

#: ../DistUpgrade/DistUpgradeFetcherCore.py:110
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:110
#, python-format
msgid "extracting '%s'"
msgstr ""

#: ../DistUpgrade/DistUpgradeFetcherCore.py:130
#: ../DistUpgrade/DistUpgradeFetcherCore.py:131
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:130
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:131
msgid "Could not run the upgrade tool"
msgstr "Gat ekki keyrt uppfærslutólið"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:131
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:131
msgid ""
"This is most likely a bug in the upgrade tool. Please report it as a bug "
"using the command 'ubuntu-bug update-manager'."
msgstr ""

#: ../DistUpgrade/DistUpgradeFetcherCore.py:206
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:206
msgid "Upgrade tool signature"
msgstr "Auðkenni uppfærslutóls"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:213
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:213
msgid "Upgrade tool"
msgstr "Uppfærsluforrit"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:247
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:247
msgid "Failed to fetch"
msgstr "Ekki tókst að sækja"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:248
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:248
msgid "Fetching the upgrade failed. There may be a network problem. "
msgstr ""
"Það tókst ekki að sækja uppfærsluna. Þetta gæti verið vandamál með netið. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:252
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:252
msgid "Authentication failed"
msgstr "Auðkenning mistókst"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:253
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:253
msgid ""
"Authenticating the upgrade failed. There may be a problem with the network "
"or with the server. "
msgstr ""
"Það tókst ekki að auðkenna uppfærsluna. Þetta er mögulega vandamál við "
"nettenginguna eða netþjóninn. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:258
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:258
msgid "Failed to extract"
msgstr "Gat ekki afþjappað"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:259
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:259
msgid ""
"Extracting the upgrade failed. There may be a problem with the network or "
"with the server. "
msgstr ""
"Það mistókst að nálgast upplýsingarnar um uppfærsluna. Þetta er mögulega "
"vandamál við nettenginguna eða netþjóninn. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:264
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:264
msgid "Verification failed"
msgstr "Staðfesting mistókst"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:265
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:265
msgid ""
"Verifying the upgrade failed.  There may be a problem with the network or "
"with the server. "
msgstr ""
"Það tókst ekki að staðfesta uppfærsluna. Það gæti verið vandamál með "
"nettenginguna eða netþjóninn. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:279
#: ../DistUpgrade/DistUpgradeFetcherCore.py:285
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:279
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:285
msgid "Can not run the upgrade"
msgstr "Get ekki keyrt uppfærsluna"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:280
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:280
msgid ""
"This usually is caused by a system where /tmp is mounted noexec. Please "
"remount without noexec and run the upgrade again."
msgstr ""

#: ../DistUpgrade/DistUpgradeFetcherCore.py:286
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:286
#, python-format
msgid "The error message is '%s'."
msgstr "Villuskilaboðið er ‚%s‘."

#: ../DistUpgrade/DistUpgradeViewText.py:91
msgid ""
"Please report this as a bug and include the files /var/log/dist-"
"upgrade/main.log and /var/log/dist-upgrade/apt.log in your report. The "
"upgrade has aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""

#: ../DistUpgrade/DistUpgradeViewText.py:115
msgid "Aborting"
msgstr "Hætti við"

#: ../DistUpgrade/DistUpgradeViewText.py:120
msgid "Demoted:\n"
msgstr "Lækkað í tign:\n"

#: ../DistUpgrade/DistUpgradeViewText.py:127
msgid "To continue please press [ENTER]"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewText.py:155
#: ../DistUpgrade/DistUpgradeViewText.py:193
#: ../DistUpgrade/DistUpgradeViewText.py:200
msgid "Continue [yN] "
msgstr "Viltu halda áfram [jN] "

#: ../DistUpgrade/DistUpgradeViewText.py:155
#: ../DistUpgrade/DistUpgradeViewText.py:193
msgid "Details [d]"
msgstr "Frekari upplýsingar [u]"

#. TRANSLATORS: the "y" is "yes"
#. TRANSLATORS: first letter of a positive (yes) answer
#: ../DistUpgrade/DistUpgradeViewText.py:159
#: ../DistUpgrade/DistUpgradeViewText.py:203
msgid "y"
msgstr "j"

#. TRANSLATORS: the "n" is "no"
#. TRANSLATORS: first letter of a negative (no) answer
#: ../DistUpgrade/DistUpgradeViewText.py:162
#: ../DistUpgrade/DistUpgradeViewText.py:210
msgid "n"
msgstr "n"

#. TRANSLATORS: the "d" is "details"
#: ../DistUpgrade/DistUpgradeViewText.py:165
msgid "d"
msgstr "u"

#: ../DistUpgrade/DistUpgradeViewText.py:170
#, python-format
msgid "No longer supported: %s\n"
msgstr "Ekki lengur stutt: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:175
#, python-format
msgid "Remove: %s\n"
msgstr "Fjarlægja: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:185
#, python-format
msgid "Install: %s\n"
msgstr "Setja upp: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:190
#, python-format
msgid "Upgrade: %s\n"
msgstr "Uppfæra: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:207
msgid "Continue [Yn] "
msgstr "Viltu halda áfram [Jn] "

#: ../DistUpgrade/DistUpgradeViewText.py:228
msgid ""
"To finish the upgrade, a restart is required.\n"
"If you select 'y' the system will be restarted."
msgstr ""
"Til að ljúka við uppfærsluna þarf að endurræsa tölvuna.\n"
"Ef þú velur ‚j‘ mun hún endurræsa sig."

#: ../DistUpgrade/DistUpgrade.ui.h:1
msgid "_Cancel Upgrade"
msgstr "_Hætta við uppfærslu"

#: ../DistUpgrade/DistUpgrade.ui.h:2
msgid "_Resume Upgrade"
msgstr "_Halda uppfærslu áfram"

#: ../DistUpgrade/DistUpgrade.ui.h:3
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly adviced to resume the upgrade."
msgstr ""
"<b><big>Hætta við þá uppfærslu sem er í gangi?</big></b>\n"
" \n"
"Kerfið gæti orðið ónothæft ef þú hættir við uppfærlsuna núna. Það er "
"stranglega mælt með að þú haldir áfram með uppfærsluna."

#: ../DistUpgrade/DistUpgrade.ui.h:6
msgid "_Start Upgrade"
msgstr "_Hefja uppfærslu"

#: ../DistUpgrade/DistUpgrade.ui.h:9
msgid "_Replace"
msgstr "_Skipta út"

#: ../DistUpgrade/DistUpgrade.ui.h:10
msgid "Difference between the files"
msgstr "Munur á skránum"

#: ../DistUpgrade/DistUpgrade.ui.h:11
msgid "_Report Bug"
msgstr "_Tilkynna villu"

#: ../DistUpgrade/DistUpgrade.ui.h:12
msgid "_Continue"
msgstr "_Halda áfram"

#: ../DistUpgrade/DistUpgrade.ui.h:13
msgid "<b><big>Start the upgrade?</big></b>"
msgstr "<b><big>Hefja uppfærslu?</big></b>"

#: ../DistUpgrade/DistUpgrade.ui.h:15
msgid ""
"<b><big>Restart the system to complete the upgrade</big></b>\n"
"\n"
"Please save your work before continuing."
msgstr ""
"<b><big>Endurræstu tölvuna til að ljúka uppfærslunni</big></b>\n"
" \n"
"Mundu eftir því að vista öll gögn áður en þú heldur áfram."

#: ../DistUpgrade/DistUpgrade.ui.h:18
msgid "Distribution Upgrade"
msgstr "Kerfisuppfærsla"

#: ../DistUpgrade/DistUpgrade.ui.h:19
msgid "<b><big>Upgrading Ubuntu to version 12.04</big></b>"
msgstr ""

#: ../DistUpgrade/DistUpgrade.ui.h:20
msgid "    "
msgstr "    "

#: ../DistUpgrade/DistUpgrade.ui.h:22
msgid "Setting new software channels"
msgstr "Ákveð nýjar hugbúnaðarrásir"

#: ../DistUpgrade/DistUpgrade.ui.h:24
msgid "Restarting the computer"
msgstr "Endurræsi tölvuna"

#: ../DistUpgrade/DistUpgrade.ui.h:27
msgid "Terminal"
msgstr "Útstöð"

#: ../UpdateManager/backend/InstallBackendSynaptic.py:60
msgid "Please wait, this can take some time."
msgstr "Vinsamlegast dokaðu, þetta gæti tekið drjúga stund."

#: ../UpdateManager/backend/InstallBackendSynaptic.py:62
msgid "Update is complete"
msgstr "Uppfærslu er lokið"

#: ../UpdateManager/DistUpgradeFetcher.py:108
#: ../UpdateManager/DistUpgradeFetcherKDE.py:103
msgid "Could not find the release notes"
msgstr "Fann ekki upplýsingar um útgáfu"

#: ../UpdateManager/DistUpgradeFetcher.py:109
#: ../UpdateManager/DistUpgradeFetcherKDE.py:104
msgid "The server may be overloaded. "
msgstr "Það er mögulega mikið álag á netþjóninum. "

#: ../UpdateManager/DistUpgradeFetcher.py:119
#: ../UpdateManager/DistUpgradeFetcherKDE.py:108
msgid "Could not download the release notes"
msgstr "Gat ekki sótt upplýsingar um útgáfuna"

#: ../UpdateManager/DistUpgradeFetcher.py:120
#: ../UpdateManager/DistUpgradeFetcherKDE.py:109
msgid "Please check your internet connection."
msgstr "Vinsamlegast athugaðu nettenginguna þína"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:62
#: ../UpdateManager/DistUpgradeFetcherKDE.py:85
msgid "Upgrade"
msgstr "Uppfæra"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:89
#: ../data/gtkbuilder/UpdateManager.ui.h:20
#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:2
msgid "Release Notes"
msgstr "Upplýsingar um útgáfu"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:128
#: ../UpdateManager/DistUpgradeFetcherKDE.py:141
#: ../UpdateManager/DistUpgradeFetcherKDE.py:143
msgid "Downloading additional package files..."
msgstr "Sæki aukapakka..."

#: ../UpdateManager/DistUpgradeFetcherKDE.py:141
#, python-format
msgid "File %s of %s at %sB/s"
msgstr "Skrá %s af %s á %sB/sek."

#: ../UpdateManager/DistUpgradeFetcherKDE.py:143
#, python-format
msgid "File %s of %s"
msgstr "Skrá %s af %s"

#: ../UpdateManager/ChangelogViewer.py:73
msgid "Open Link in Browser"
msgstr "Opna tengil í vafra"

#: ../UpdateManager/ChangelogViewer.py:76
msgid "Copy Link to Clipboard"
msgstr "Afrita tengil á klippiborð"

#: ../UpdateManager/GtkProgress.py:159
#, python-format
msgid "Downloading file %(current)li of %(total)li with %(speed)s/s"
msgstr "Sæki skrá %(current)li af %(total)li á %(speed)s/sek."

#: ../UpdateManager/GtkProgress.py:164
#, python-format
msgid "Downloading file %(current)li of %(total)li"
msgstr "Sæki skrá %(current)li af %(total)li"

#: ../UpdateManager/UpdateManager.py:104 ../do-release-upgrade:95
msgid "Your Ubuntu release is not supported anymore."
msgstr ""

#: ../UpdateManager/UpdateManager.py:105
msgid ""
"You will not get any further security fixes or critical updates. Please "
"upgrade to a later version of Ubuntu."
msgstr ""

#: ../UpdateManager/UpdateManager.py:113
msgid "Upgrade information"
msgstr "Upplýsingar um uppfærslu"

#: ../UpdateManager/UpdateManager.py:229
#: ../UpdateManagerText/UpdateManagerText.py:32
msgid "Install"
msgstr "Setja upp"

#: ../UpdateManager/UpdateManager.py:231
msgid "Name"
msgstr ""

#. upload_archive = version_match.group(2).strip()
#: ../UpdateManager/UpdateManager.py:391
#, python-format
msgid "Version %s: \n"
msgstr "Útgáfa %s: \n"

#: ../UpdateManager/UpdateManager.py:453
msgid ""
"No network connection detected, you can not download changelog information."
msgstr ""
"Engin nettenging fannst, þú getur ekki sótt upplýsingar varðandi "
"breytingalista."

#: ../UpdateManager/UpdateManager.py:461
msgid "Downloading list of changes..."
msgstr "Sæki lista yfir breytingar..."

#: ../UpdateManager/UpdateManager.py:505
msgid "_Deselect All"
msgstr ""

#: ../UpdateManager/UpdateManager.py:511
msgid "Select _All"
msgstr ""

#: ../UpdateManager/UpdateManager.py:571
#, python-format
msgid "%(count)s update has been selected."
msgid_plural "%(count)s updates have been selected."
msgstr[0] ""
msgstr[1] ""

#: ../UpdateManager/UpdateManager.py:575
#, python-format
msgid "%s will be downloaded."
msgstr ""

#: ../UpdateManager/UpdateManager.py:587
msgid "The update has already been downloaded, but not installed."
msgid_plural "The updates have already been downloaded, but not installed."
msgstr[0] "Uppfærslan hefur þegar verið sótt, en ekki sett upp."
msgstr[1] "Uppfærslurnar hafa þegar verið sóttar, en ekki settar upp."

#: ../UpdateManager/UpdateManager.py:592
msgid "There are no updates to install."
msgstr "Það eru engar uppfærslur til að setja upp."

#. TRANSLATORS: this allows to switch the order of the count of
#. updates and the download size string (if needed)
#: ../UpdateManager/UpdateManager.py:598
#, python-format
msgid "%(count_str)s %(download_str)s"
msgstr "%(count_str)s %(download_str)s"

#: ../UpdateManager/UpdateManager.py:605
msgid "Unknown download size."
msgstr "Óþekkt stærð á niðurhali."

#: ../UpdateManager/UpdateManager.py:631
msgid ""
"It is unknown when the package information was updated last. Please click "
"the 'Check' button to update the information."
msgstr ""
"Það er ekki vitað hvenær pakkaupplýsingarnar voru uppfærðar síðast. "
"Vinsamlegast smelltu á hnappann 'Athuga' til að uppfæra upplýsingarnar."

#: ../UpdateManager/UpdateManager.py:637
#, python-format
msgid ""
"The package information was last updated %(days_ago)s days ago.\n"
"Press the 'Check' button below to check for new software updates."
msgstr ""
"Pakkaupplýsingarnar voru síðast uppfærðar fyrir %(days_ago)s dögum síðan.\n"
"Smelltu á hnappann 'Athuga' fyrir neðan til að athuga með nýjar "
"hugbúnaðaruppfærslur.."

#: ../UpdateManager/UpdateManager.py:642
#, python-format
msgid "The package information was last updated %(days_ago)s day ago."
msgid_plural ""
"The package information was last updated %(days_ago)s days ago."
msgstr[0] ""
"Pakkaupplýsingarnar voru síðast uppfærðar fyrir %(days_ago)s degi."
msgstr[1] ""
"Pakkaupplýsingarnar voru síðast uppfærðar fyrir %(days_ago)s dögum."

#: ../UpdateManager/UpdateManager.py:646
#, python-format
msgid "The package information was last updated %(hours_ago)s hour ago."
msgid_plural ""
"The package information was last updated %(hours_ago)s hours ago."
msgstr[0] ""
msgstr[1] ""

#. TRANSLATORS: only in plural form, as %s minutes ago is one of 15, 30, 45 minutes ago
#: ../UpdateManager/UpdateManager.py:651 ../UpdateManager/UpdateManager.py:653
#: ../UpdateManager/UpdateManager.py:655
#, python-format
msgid "The package information was last updated about %s minutes ago."
msgstr "Pakkaupplýsingarnar voru síðast uppfærðar fyrir um %s mínútum."

#: ../UpdateManager/UpdateManager.py:657
msgid "The package information was just updated."
msgstr "Það var verið að uppfæra pakkaupplýsingarnar."

#: ../UpdateManager/UpdateManager.py:677
#: ../data/gtkbuilder/UpdateManager.ui.h:2
msgid ""
"Software updates correct errors, eliminate security vulnerabilities and "
"provide new features."
msgstr ""
"Hugbúnaðauppfærslur leiðrétta villur, lagfæra öryggisveilur og innleiða nýja "
"möguleika."

#: ../UpdateManager/UpdateManager.py:696
msgid "Software updates may be available for your computer."
msgstr "Hugbúnaðaruppfærslur gætu verið fáanlegar fyrir tölvuna þína."

#: ../UpdateManager/UpdateManager.py:704
msgid "Welcome to Ubuntu"
msgstr "Velkomin í Ubuntu"

#: ../UpdateManager/UpdateManager.py:705
msgid ""
"These software updates have been issued since this version of Ubuntu was "
"released."
msgstr ""
"Þessar hugbúnaðaruppfærslur hafa verið gefnar út síðan þessi útgáfa af "
"Ubuntu var gefin út."

#: ../UpdateManager/UpdateManager.py:708
msgid "Software updates are available for this computer."
msgstr "Hugbúnaðaruppfærslur eru fáanlegar fyrir þessa tölvu."

#: ../UpdateManager/UpdateManager.py:759
#, python-format
msgid ""
"The upgrade needs a total of %s free space on disk '%s'. Please free at "
"least an additional %s of disk space on '%s'. Empty your trash and remove "
"temporary packages of former installations using 'sudo apt-get clean'."
msgstr ""
"Uppfærslan krefst %s af plássi á diskinum ‚%s‘. Reyndu að rýma til alla "
"vegana %s af plássi á ‚%s‘. Prufaðu að tæma ruslið og að fjarlægja "
"bráðabirgðapakka sem gamlar uppsetningar skildu eftir sig með því að skrifa "
"‚sudo apt-get clean‘ í útstöðina."

#: ../UpdateManager/UpdateManager.py:784
msgid ""
"The computer needs to restart to finish installing updates. Please save your "
"work before continuing."
msgstr ""
"Það þarf að endurræsa tölvuna til að ljúka uppfærslum. Vistaðu öll skjöl "
"áður en haldið er áfram."

#: ../UpdateManager/UpdateManager.py:848
msgid "Reading package information"
msgstr "Les pakkaupplýsingar"

#: ../UpdateManager/UpdateManager.py:863
msgid "Connecting..."
msgstr "Tengist..."

#: ../UpdateManager/UpdateManager.py:880
msgid "You may not be able to check for updates or download new updates."
msgstr ""
"Það gæti verið að þú getir ekki athugað með uppfærslur eða sótt nýjar "
"uppfærslur."

#: ../UpdateManager/UpdateManager.py:1004
msgid "Could not initialize the package information"
msgstr "Gat ekki frumstillt pakkaupplýsingar"

#: ../UpdateManager/UpdateManager.py:1005
msgid ""
"An unresolvable problem occurred while initializing the package "
"information.\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"following error message:\n"
msgstr ""

#: ../UpdateManager/UpdateManager.py:1036
msgid ""
"An unresolvable problem occurred while calculating the upgrade.\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"following error message:"
msgstr ""

#: ../UpdateManager/UpdateManager.py:1061
msgid " (New install)"
msgstr " (ný uppsetning)"

#. TRANSLATORS: the b stands for Bytes
#: ../UpdateManager/UpdateManager.py:1068
#, python-format
msgid "(Size: %s)"
msgstr "(stærð: %s)"

#: ../UpdateManager/UpdateManager.py:1070
#, python-format
msgid "From version %(old_version)s to %(new_version)s"
msgstr "Frá útgáfu %(old_version)s upp í %(new_version)s"

#: ../UpdateManager/UpdateManager.py:1074
#, python-format
msgid "Version %s"
msgstr "Útgáfa %s"

#: ../UpdateManager/UpdateManager.py:1105 ../do-release-upgrade:107
msgid "Release upgrade not possible right now"
msgstr "Útgáfuuppfærsla ekki möguleg í augnablikinu."

#: ../UpdateManager/UpdateManager.py:1106 ../do-release-upgrade:108
#, c-format, python-format
msgid ""
"The release upgrade can not be performed currently, please try again later. "
"The server reported: '%s'"
msgstr ""

#: ../UpdateManager/UpdateManager.py:1108 ../check-new-release-gtk:115
msgid "Downloading the release upgrade tool"
msgstr "Sækir uppfærslutólið fyrir útgáfuna"

#: ../UpdateManager/UpdateManager.py:1115
#, python-format
msgid "<b>New Ubuntu release '%s' is available</b>"
msgstr "<b>Nýja ‚%s‘ Ubuntu-útgáfan er fáanleg</b>"

#. we assert a clean cache
#: ../UpdateManager/UpdateManager.py:1154
msgid "Software index is broken"
msgstr "Hugbúnaðarvísirinn er bilaður"

#: ../UpdateManager/UpdateManager.py:1155
msgid ""
"It is impossible to install or remove any software. Please use the package "
"manager \"Synaptic\" or run \"sudo apt-get install -f\" in a terminal to fix "
"this issue at first."
msgstr ""
"Það er ómögulegt að setja upp eða fjarlægja nokkurn hugbúnað. Notaðu "
"„Synaptic‟-pakkastjórann eða keyrðu skipunina „sudo apt-get install -f‟ í "
"útstöðina til að laga þetta vandamál."

#: ../UpdateManager/UnitySupport.py:57
msgid "Check for Updates"
msgstr ""

#: ../UpdateManager/UnitySupport.py:66
msgid "Install All Available Updates"
msgstr ""

#: ../UpdateManagerText/UpdateManagerText.py:31
msgid "Cancel"
msgstr "Hætta við"

#: ../UpdateManagerText/UpdateManagerText.py:34
msgid "Changelog"
msgstr ""

#: ../UpdateManagerText/UpdateManagerText.py:37
msgid "Updates"
msgstr ""

#: ../UpdateManagerText/UpdateManagerText.py:50
msgid "Building Updates List"
msgstr "Bý til lista yfir uppfærslur"

#: ../UpdateManagerText/UpdateManagerText.py:53
msgid ""
"\n"
"A normal upgrade can not be calculated, please run: \n"
"  sudo apt-get dist-upgrade\n"
"\n"
"\n"
"This can be caused by:\n"
" * A previous upgrade which didn't complete\n"
" * Problems with some of the installed software\n"
" * Unofficial software packages not provided by Ubuntu\n"
" * Normal changes of a pre-release version of Ubuntu"
msgstr ""
"\n"
"Ekki var mögulegt að reikna venjulega uppfærslu, vinsamlegast keyrið: \n"
"  sudo apt-get dist-upgrade\n"
"\n"
"\n"
"Örsakavaldur getur verið eftirfarandi:\n"
" * Fyrri uppfærsla kláraði ekki sem skildi\n"
" * Vandamál í einhverjum hugbúnaði\n"
" * Óformlegir hugbúnaðarpakkar sem ekki er á vegum Ubuntu\n"
" * Venjulegar breytingar á prufuútgáfu Ubuntu"

#: ../UpdateManagerText/UpdateManagerText.py:122
msgid "Downloading changelog"
msgstr "Sæki breytingarskrá"

#: ../UpdateManager/Core/MyCache.py:138
#, python-format
msgid "Other updates (%s)"
msgstr "Aðrar uppfærslur (%s)"

#: ../UpdateManager/Core/MyCache.py:316
msgid "This update does not come from a source that supports changelogs."
msgstr ""

#: ../UpdateManager/Core/MyCache.py:322 ../UpdateManager/Core/MyCache.py:350
msgid ""
"Failed to download the list of changes. \n"
"Please check your Internet connection."
msgstr ""
"Það tókst ekki að sækja lista yfir breytingar. \n"
"Athugaðu með nettenginguna þína."

#: ../UpdateManager/Core/MyCache.py:329
#, python-format
msgid ""
"Changes for the versions:\n"
"Installed version: %s\n"
"Available version: %s\n"
"\n"
msgstr ""

#: ../UpdateManager/Core/MyCache.py:339
#, python-format
msgid ""
"The changelog does not contain any relevant changes.\n"
"\n"
"Please use http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"until the changes become available or try again later."
msgstr ""

#: ../UpdateManager/Core/MyCache.py:344
#, python-format
msgid ""
"The list of changes is not available yet.\n"
"\n"
"Please use http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"until the changes become available or try again later."
msgstr ""
"Listi yfir breytingar er ekki fáanlegur enn sem komið er.\n"
"\n"
"Notaðu http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"þangað til breytingarnar eru fáanlegar eða reyndu aftur síðar."

#: ../UpdateManager/Core/UpdateList.py:49
msgid "Failed to detect distribution"
msgstr "Gat ekki komist að því hvaða útgáfa þetta er"

#: ../UpdateManager/Core/UpdateList.py:50
#, python-format
msgid "A error '%s' occurred while checking what system you are using."
msgstr "Villan ‚%s‘ kom upp þegar tölvan var að kanna hvaða kerfi þú notar."

#: ../UpdateManager/Core/UpdateList.py:61
msgid "Important security updates"
msgstr "Mikilvægar öryggisuppfærslur"

#: ../UpdateManager/Core/UpdateList.py:62
msgid "Recommended updates"
msgstr "Ráðlagðar uppfærslur"

#: ../UpdateManager/Core/UpdateList.py:63
msgid "Proposed updates"
msgstr "Fyrirhugaðar uppfærslur"

#: ../UpdateManager/Core/UpdateList.py:64
msgid "Backports"
msgstr "Endurgerð forrit"

#: ../UpdateManager/Core/UpdateList.py:65
msgid "Distribution updates"
msgstr "Uppfærslur fyrir dreifinguna"

#: ../UpdateManager/Core/UpdateList.py:70
msgid "Other updates"
msgstr "Aðrar uppfærslur"

#: ../data/gtkbuilder/UpdateManager.ui.h:1
msgid "<big><b>Starting Update Manager</b></big>"
msgstr "<big><b>Ræsi uppfærslustjórann</b></big>"

#: ../data/gtkbuilder/UpdateManager.ui.h:3
msgid "_Partial Upgrade"
msgstr "_Hlutauppfærsla"

#: ../data/gtkbuilder/UpdateManager.ui.h:4
msgid "<big><b>Not all updates can be installed</b></big>"
msgstr "<big><b>Það tókst ekki að segja upp allar uppfærslurnar</b></big>"

#: ../data/gtkbuilder/UpdateManager.ui.h:5
msgid ""
"Run a partial upgrade, to install as many updates as possible. \n"
"\n"
"This can be caused by:\n"
" * A previous upgrade which didn't complete\n"
" * Problems with some of the installed software\n"
" * Unofficial software packages not provided by Ubuntu\n"
" * Normal changes of a pre-release version of Ubuntu"
msgstr ""

#: ../data/gtkbuilder/UpdateManager.ui.h:12
msgid "Chec_k"
msgstr "_Athuga"

#: ../data/gtkbuilder/UpdateManager.ui.h:13
msgid ""
"<b><big>You must check for updates manually</big></b>\n"
"\n"
"Your system does not check for updates automatically. You can configure this "
"behavior in <i>Software Sources</i> on the <i>Updates</i> tab."
msgstr ""
"<b><big>Þú verður að leita að uppfærslum í höndunum</big></b>\n"
"\n"
"Tölvan þín leitar ekki að uppfærslum sjálfkrafa. Þú getur breytt þessu í "
"<i>Hugbúnaðarupptökum</i> undir flipanum <i>Uppfærslur</i>."

#: ../data/gtkbuilder/UpdateManager.ui.h:16
msgid "_Hide this information in the future"
msgstr "_Fela þessar upplýsingar í framtíðinni"

#: ../data/gtkbuilder/UpdateManager.ui.h:17
msgid "Co_ntinue"
msgstr "_Halda áfram"

#: ../data/gtkbuilder/UpdateManager.ui.h:18
msgid "<big><b>Running on battery</b></big>"
msgstr "<big><b>Nota rafhlöðu</b></big>"

#: ../data/gtkbuilder/UpdateManager.ui.h:19
msgid "Your system is running on battery. Are you sure you want to continue?"
msgstr "Tölvan þín notar rafhlöðu. Viltu örugglega halda áfram?"

#: ../data/gtkbuilder/UpdateManager.ui.h:21
msgid "_Upgrade"
msgstr "_Uppfæra"

#: ../data/gtkbuilder/UpdateManager.ui.h:22
#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:8
msgid "Show progress of individual files"
msgstr "Skoða framvindu einstakra skráa"

#: ../data/gtkbuilder/UpdateManager.ui.h:23
#: ../data/update-manager.desktop.in.h:1
msgid "Update Manager"
msgstr "Uppfærslustjóri"

#: ../data/gtkbuilder/UpdateManager.ui.h:24
msgid "Starting Update Manager"
msgstr ""

#: ../data/gtkbuilder/UpdateManager.ui.h:25
msgid "U_pgrade"
msgstr "_Uppfæra"

#: ../data/gtkbuilder/UpdateManager.ui.h:27
msgid "updates"
msgstr "uppfærslur"

#: ../data/gtkbuilder/UpdateManager.ui.h:28
msgid ""
"You are connected via roaming and may be charged for the data consumed by "
"this update."
msgstr ""

#: ../data/gtkbuilder/UpdateManager.ui.h:29
msgid "You are connected via a wireless modem."
msgstr ""

#: ../data/gtkbuilder/UpdateManager.ui.h:30
msgid "It’s safer to connect the computer to AC power before updating."
msgstr ""

#: ../data/gtkbuilder/UpdateManager.ui.h:31
msgid "_Install Updates"
msgstr "_Setja upp uppfærslur"

#: ../data/gtkbuilder/UpdateManager.ui.h:32
msgid "Changes"
msgstr "Breytingar"

#: ../data/gtkbuilder/UpdateManager.ui.h:33
msgid "Description"
msgstr "Lýsing"

#: ../data/gtkbuilder/UpdateManager.ui.h:34
msgid "Description of update"
msgstr "Uppfærslulýsing"

#: ../data/gtkbuilder/UpdateManager.ui.h:35
msgid "_Settings..."
msgstr ""

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:1
msgid ""
"<b>A new version of Ubuntu is available. Would you like to upgrade?</b>"
msgstr "<b>Ný útgáfa Ubuntu er tiltæk. Viltu uppfæra í hana núna?</b>"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:3
msgid "Don't Upgrade"
msgstr "Ekki uppfæra"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:4
msgid "Ask Me Later"
msgstr "Spyrja síðar"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:5
msgid "Yes, Upgrade Now"
msgstr "Já, uppfæra núna"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:6
msgid "You have declined to upgrade to the new Ubuntu"
msgstr "Þú hefur hafnað uppfærslu í nýja útgáfu af Ubuntu"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:7
msgid ""
"You can upgrade at a later time by opening Update Manager and click on "
"\"Upgrade\"."
msgstr ""
"Þú getur uppfært síðar með því að opna Uppfærslustjórann og smella á "
"\"Uppfæra\"."

#: ../data/update-manager.desktop.in.h:2
msgid "Software Updates"
msgstr "Hugbúnaðaruppfærslur"

#: ../data/update-manager.desktop.in.h:3
msgid "Show and install available updates"
msgstr "Sýna og setja upp uppfærslur"

#: ../update-manager:61 ../update-manager-text:50 ../do-release-upgrade:46
msgid "Show version and exit"
msgstr "Sýna útgáfu og hætta"

#: ../update-manager:64
msgid "Directory that contains the data files"
msgstr "Mappan sem inniheldur gagnaskrárnar"

#: ../update-manager:67
msgid "Check if a new Ubuntu release is available"
msgstr "Athuga hvort til sé ný útgáfa af Ubuntu"

#: ../update-manager:70 ../do-release-upgrade:49 ../check-new-release-gtk:182
msgid "Check if upgrading to the latest devel release is possible"
msgstr "Athuga hvort hægt sé að uppfæra í nýjustu þróunarútgáfuna"

#: ../update-manager:74
msgid "Upgrade using the latest proposed version of the release upgrader"
msgstr ""

#: ../update-manager:81
msgid "Do not focus on map when starting"
msgstr ""

#: ../update-manager:84
msgid "Try to run a dist-upgrade"
msgstr "Reyndu að keyra drefingaruppfærslu"

#: ../update-manager:87
msgid "Do not check for updates when starting"
msgstr ""

#: ../update-manager:91 ../do-release-upgrade:65
msgid "Test upgrade with a sandbox aufs overlay"
msgstr ""

#: ../update-manager:111
msgid "Running partial upgrade"
msgstr "Keyra hlutauppfærslu"

#: ../update-manager-text:54
msgid "Show description of the package instead of the changelog"
msgstr ""

#: ../do-release-upgrade:53 ../check-new-release-gtk:186
msgid ""
"Try upgrading to the latest release using the upgrader from $distro-proposed"
msgstr ""

#: ../do-release-upgrade:57
msgid ""
"Run in a special upgrade mode.\n"
"Currently 'desktop' for regular upgrades of a desktop system and 'server' "
"for server systems are supported."
msgstr ""

#: ../do-release-upgrade:63
msgid "Run the specified frontend"
msgstr "Keyra tiltekið viðmót"

#: ../do-release-upgrade:68
msgid ""
"Check only if a new distribution release is available and report the result "
"via the exit code"
msgstr ""

#: ../do-release-upgrade:82
msgid "Checking for a new Ubuntu release"
msgstr ""

#: ../do-release-upgrade:96
msgid ""
"For upgrade information, please visit:\n"
"%(url)s\n"
msgstr ""

#: ../do-release-upgrade:102
msgid "No new release found"
msgstr "Engar nýjar útgáfur fundust"

#: ../do-release-upgrade:114
#, c-format
msgid "New release '%s' available."
msgstr "Nýja útgáfan ‚%s‘ fannst."

#: ../do-release-upgrade:115
msgid "Run 'do-release-upgrade' to upgrade to it."
msgstr "Keyrðu ‚do-release-upgrade‘ til að hefja uppfærsluna."

#: ../check-new-release-gtk:90
msgid "Ubuntu %(version)s Upgrade Available"
msgstr "Uppfærsla í Ubuntu %(version)s er tiltæk"

#: ../check-new-release-gtk:132
#, c-format
msgid "You have declined the upgrade to Ubuntu %s"
msgstr "Þú hefur hafnað uppfærslu í Ubuntu %s"

#: ../check-new-release-gtk:192
msgid "Add debug output"
msgstr ""

#: ../ubuntu-support-status:79
msgid "Show unsupported packages on this machine"
msgstr ""

#: ../ubuntu-support-status:82
msgid "Show supported packages on this machine"
msgstr ""

#: ../ubuntu-support-status:85
msgid "Show all packages with their status"
msgstr ""

#: ../ubuntu-support-status:88
msgid "Show all packages in a list"
msgstr ""

#: ../ubuntu-support-status:130
#, c-format
msgid "Support status summary of '%s':"
msgstr ""

#: ../ubuntu-support-status:133
msgid "You have %(num)s packages (%(percent).1f%%) supported until %(time)s"
msgstr ""

#: ../ubuntu-support-status:139
msgid ""
"You have %(num)s packages (%(percent).1f%%) that can not/no-longer be "
"downloaded"
msgstr ""

#: ../ubuntu-support-status:142
msgid "You have %(num)s packages (%(percent).1f%%) that are unsupported"
msgstr ""

#: ../ubuntu-support-status:150
msgid ""
"Run with --show-unsupported, --show-supported or --show-all to see more "
"details"
msgstr ""

#: ../ubuntu-support-status:154
msgid "No longer downloadable:"
msgstr ""

#: ../ubuntu-support-status:157
msgid "Unsupported: "
msgstr ""

#: ../ubuntu-support-status:162
#, c-format
msgid "Supported until %s:"
msgstr ""

#: ../ubuntu-support-status:171
msgid "Unsupported"
msgstr ""

#: ../Janitor/computerjanitor/exc.py:30
#, python-format
msgid "Unimplemented method: %s"
msgstr "Aðferð sem er ekki notuð: %s"

#: ../Janitor/computerjanitor/file_cruft.py:46
msgid "A file on disk"
msgstr "Skrá á diski"

#: ../Janitor/computerjanitor/package_cruft.py:45
msgid ".deb package"
msgstr ".deb-pakki"

#: ../Janitor/computerjanitor/missing_package_cruft.py:33
msgid "Install missing package."
msgstr "Setja upp pakkann sem vantar."

#: ../Janitor/computerjanitor/missing_package_cruft.py:42
#, python-format
msgid "Package %s should be installed."
msgstr "Það ætti að setja upp pakkann %s."

#. pragma: no cover
#: ../Janitor/plugins/dpkg_status_plugin.py:36
#, python-format
msgid "%i obsolete entries in the status file"
msgstr ""

#: ../Janitor/plugins/dpkg_status_plugin.py:39
msgid "Obsolete entries in dpkg status"
msgstr ""

#. pragma: no cover
#: ../Janitor/plugins/dpkg_status_plugin.py:42
msgid "Obsolete dpkg status entries"
msgstr ""

#: ../Janitor/plugins/kdelibs4to5_plugin.py:39
msgid ""
"When upgrading, if kdelibs4-dev is installed, kdelibs5-dev needs to be "
"installed. See bugs.launchpad.net, bug #279621 for details."
msgstr ""

#: ../Janitor/plugins/langpack_manual_plugin.py:34
#, python-format
msgid "%s needs to be marked as manually installed."
msgstr ""

#: ../Janitor/plugins/remove_lilo_plugin.py:28
msgid ""
"Remove lilo since grub is also installed.(See bug #314004 for details.)"
msgstr ""
"Fjarlægja lilo þar sem grub er til staðar.(See bug #314004 for details.)"

#~ msgid ""
#~ "If you don't want to install them now, choose \"Update Manager\" from the "
#~ "Administration menu later."
#~ msgstr ""
#~ "Þú getur farið í „Uppfærslustjórann“ í stjórnunarvalmyndinni seinna ef þú "
#~ "vilt ekki setja þær upp núna."

#, python-format
#~ msgid ""
#~ "This upgrade is running in sandbox (test) mode. All changes are written to "
#~ "'%s' and will be lost on the next reboot.\n"
#~ "\n"
#~ "*No* changes written to a systemdir from now until the next reboot are "
#~ "permanent."
#~ msgstr ""
#~ "Þessi uppfærsla fer fram í sandkassanum (til að testa). Allar breytingar sem "
#~ "þú gerir fara í ‚%s‘ og eyðast þegar þú kveikir næst á tölvunni.\n"
#~ "\n"
#~ "Allar breytingar sem gerðar eru á systemdir verða teknar aftur þegar þú "
#~ "kveikir næst á tölvunni."

#~ msgid ""
#~ "Fetching and installing the upgrade can take several hours. Once the "
#~ "download has finished, the process cannot be cancelled."
#~ msgstr ""
#~ "Það getur tekið nokkra tíma að uppfæra tölvuna. Ekki er hægt að hætta við "
#~ "þegar niðurhalinu er lokið."

#~ msgid "Software updates are available for this computer"
#~ msgstr "Þú getur náð í hugbúnaðaruppfærslur fyrir þessa tölvu"

#~ msgid "Your system is up-to-date"
#~ msgstr "Kerfið þitt hefur verið uppfært"

#~ msgid "Checking for a new ubuntu release"
#~ msgstr "Leita að nýrri Ubuntu-útgáfu"

#~ msgid "0 kB"
#~ msgstr "0 kB"

#~ msgid "1 kB"
#~ msgstr "1 kB"

#, python-format
#~ msgid "%.0f kB"
#~ msgstr "%.0f kB"
